task3.py

```diff
@@ -0,0 +1,30 @@
+OUTPUT_FILE = "output.csv"
+#import pandas а если я просто оберну датафреймы в функцию мне ведь не засчитают да
+
+def to_csv_file(filename, headers, rows, separator=',', new_line='\n') -> str:
+    out = ""
+    data1 = rows
+    data1.insert(0, headers)
+    for row in data1:
+        for cell in row:
+            out += cell
+            if cell != row[-1]:
+                out += separator
+        out += new_line
+    with open(filename, 'w') as file:
+        for line1 in out:
+            file.write(line1)
```

Comment:
file.write(separator.join(headers) + new_line)
for row in rows:
....file.writeline(delimiter.join(row) + new_line)